framework/merchant_api/domain/entities/coles_product_offer.py

```diff
@@ -0,0 +1,19 @@
+from typing import List, Optional
+from pydantic import BaseModel
+
+
+class ColesProductOffer(BaseModel, extra='allow'):
+    class ImageInfo(BaseModel, extra='allow'):
+        uri: str
+
+    class Pricing(BaseModel, extra='allow'):
+        now: float
+        was: float
+
+    imageUris: List[ImageInfo]
+    id: int
+    name: str
+    brand: str
+    size: str # "352g"
+    availability: bool
+    pricing: Optional[Pricing] # None if `availability=False`
```

Comment:
does the # None if `availability=False` need to be implemented before this PR goes through?
Resolve if not.

Reply:
nah, just an old note from the code, likely just a reminder i put